schema.md

```diff
@@ -0,0 +1,43 @@
+# Cómo utilizar `schema.org`
+
+Schema.org provee una colección de vocabularios compartidos 
+que puedes utilizar para que las máquinas de búsqueda puedan entender tu página web.
+
+Vamos a utilizar un ejemplo:
+
+![schema](images/schema.png)
```

Comment:
I was expecting to see a screenshot about a schema.org type of work. I think it would be helpful for people to first see an example of a work that can be marked up with schema, article or news are common enough. 

Then you can draw a correlation between the property types and actual examples from the docs because it is also important to know how to read the documentation and determine which `itemscope` you want to use.